config/projects/google-fluentd.rb

```diff
@@ -8,13 +8,13 @@
 description "Google Fluentd: A data collector for Google Cloud Logging"
 
 install_dir     "/opt/google-fluentd"
-build_version   "1.5.9"
+build_version   "1.5.9.grpc.alpha.2"
```

Comment:
Yeah, I wonder what our convention is. Maybe just build package with sub versions (1.5.9.grpc.alpha.2) locally but do not include them in the commit and keep the versions in the master branch always major (e.g. 1.5.9) ?